frontend/src/components/housing/enquiry/EnquiryForm.vue

```diff
@@ -278,6 +279,29 @@ onMounted(async () => {
         :disabled="!editable"
       />
 
+      <SectionHeader title="ATS" />
+
+      <div class="flex align-items-center">
+        <InputText
+          v-if="formRef?.values.atsClientNumber"
+          class="col-3 flex-auto"
+          name="atsClientNumber"
+          label="ATS Client #"
+          :disabled="true"
+        />
+        <Button
+          aria-label="Link to ATS"
+          class="h-2rem ml-2"
```

Comment:
Is this a potential class typo?

Reply:
it exists!: https://primeflex.org/height